werewolves/lib/ui/about_us.dart

```diff
@@ -20,6 +21,9 @@ class AboutUsPage extends StatelessWidget {
           ),
         ),
       ),
+      floatingActionButton: FloatingActionButton(onPressed: () {
```

Comment:
we can make the scaffold with a navigation bar that has back button and remove the navigation bar from `MaterialApp`

Example: https://docs.flutter.dev/cookbook/navigation/navigation-basics#interactive-example

Reply:
Not sure how to get the automatic back arrow in the AppBar, can you add this so I can see how it's done?